clients/src/test/java/org/apache/kafka/clients/consumer/KafkaConsumerTest.java

```diff
@@ -2948,6 +2948,64 @@ public void testAssignorNameConflict() {
             () -> new KafkaConsumer<>(configs, new StringDeserializer(), new StringDeserializer()));
     }
 
+    @Test
+    public void testOffsetsForTimesTimeout() {
+        final KafkaConsumer<String, String> consumer = consumerForCheckingTimeoutException();
+        assertEquals(
+            "Failed to get offsets by times in 60000ms",
+            assertThrows(org.apache.kafka.common.errors.TimeoutException.class, () -> consumer.offsetsForTimes(singletonMap(tp0, 0L))).getMessage()
```

Comment:
Could we import `TimeoutException` instead of specifying the full qualified name every time?

Reply:
we already imported java.util.concurrent.TimeoutException so can't import org.apache.kafka.common.errors.TimeoutException here.